### PR TITLE
Fix snapshotting validation and rewrite -s to --rebuild

### LINKED ISSUE #2843: Only accept integer range values for snapshotting rounds
### Expected behavior
We should accept only the integer range values for snapshotting rounds. If need to specify the full all available rounds we should use `-1` for it. 

To run full rounds snapshot we should be able to do;

```
node app.js -s -1
```

### Actual behavior
Currently we are providing different values, integer for valid rounds and non-integer for full rounds. 
https://github.com/LiskHQ/lisk/blob/cb68ebb96aacc3dccd532de6f926a1c39a2b1411/modules/loader.js#L772

e.g. Currently we can run something like this 

```
node app.js -s all
or
node app.js -s highest
or 
node app.js -s my-name
```

### Which version(s) does this affect? (Environment, OS, etc...)
all

### PR DESCRIPTION
**Pull request by michielmulders**
### What was the problem?
Wrong validation check for snapshotting (CLI params are always strings).
Rewrite `-s` option to `--rebuild` (no `-r` as that's reserved for configuring Redis).

### How did I fix it?
Updated validation.

### How to test it?
Run tests.
Run snapshotting: `NODE_ENV=test npm start -- -n testnet --rebuild 40`
Or with `-b`

### Review checklist

* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -346,31 +346,31 @@ npm start -- [options]
...
+|                                      | LISK_FORGING_WHITELIST | forging.access.whiteList | Comma separated list of IPs to enable access to forging endpoints. Format `192.168.99.100,172.169.99.77`                                                                                                                                                                                                                                         |
+| <pre nowrap>--rebuild</pre>          |                        |                          | Rebuild the chain from Genesis block up to specified round number, must be a positive integer equal to or greater than `0`. When `0` is passed, this corresponds to the inclusion of all rounds. Any other number equals to its corresponding round. Bear in mind this mode disables all the network features of the node to ensure reliability. |
```
Comment (diego-G) [inline README.md]: I'm not so sure about the wording here. I'd suggest `--validate -v`. What we do it's to process again and therefore validate every block until the typed round.

Comment (michielmulders) [reply README.md]: So, rename the functions as well: `createSnapshot` -> `rebuildChain`.
Your comment about documentation wording is correct. We both validate and rebuild up to the specified number.

**Comment by 4miners** [comment]
Files like `snapshot.json`, `pm2-snapshot.json` should also be renamed.

File: framework/src/modules/chain/submodules/loader.js [hunk 2]
```diff
@@ -741,36 +741,31 @@ __private.validateOwnChain = cb => {
  * Snapshot creation - performs rebuild of accounts states from blockchain data
```
Comment (diego-G) [inline framework/src/modules/chain/submodules/loader.js]: Please, make another round to check for `snapshot` matches across the codebase. Careful with the concept of round snapshot which is different is meant to stay.

Comment (michielmulders) [reply framework/src/modules/chain/submodules/loader.js]: Obsolete and removed (with test), thanks!

**Pull request closed without merging**